client/src/components/Search.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { TextField, Button } from "@material-ui/core";
+
+import "../styles/Search.css";
+
+const Search = ({ onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleSearch = () => {
+    onSearch(searchTerm);
+  };
+
+  const handleInputChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
+      handleSearch();
+    }
+  };
+
+  return (
+    <div className="search-container">
```

Comment:
Best to wrap this in a `form` element, same as you have with add video. It's better for usability, if the user presses enter while in the search input, it will submit the form.